daffodil-core/src/main/scala/org/apache/daffodil/dsom/Nesting.scala

```diff
@@ -45,7 +45,10 @@ trait NestingMixin {
    *  referenced from a element ref or group ref, is NOT the ref object, but the
    *  component that contains the ref object
    */
-  final lazy val enclosingComponent: Option[SchemaComponent] = enclosingComponentDef
+  final lazy val enclosingComponent: Option[SchemaComponent] = {
```

Comment:
Unless I'm missing something, this seems like an unnecessary change.

Reply:
I can revert this. This is just giving a place for a breakpoint where you can examine the value. Very annoying to me that you can't see the value before an expression returns without permuting the code like this.